TS01-Time-Series_Metadata.Rmd

```diff
@@ -65,37 +60,20 @@ refresher on setting the working directory, please view the <a href="XXX" target
 
 </div>
 
-#Understand Your Data 
```

Comment:
@mguarinello I was wondering why you chose to shorten/ remove much of this information?  Did you feel it was too repetitive from Lesson 00?  

I too am finding the balance between flow from one lesson to another and stand alone understanding a challenge.  The thing to keep in mind is that when teaching this as a full workshop the instructor can easily skip the redundant parts.